corehq/apps/app_manager/dbaccessors.py

```diff
@@ -176,14 +177,20 @@ def get_app_cached(domain, app_id):
     """Cached version of ``get_app`` for use in phone
     api calls where most requests will be for app builds
     which are read-only.
-
     This only caches app builds."""
+    timeout = 24 * 3600
+    # cache to save in localmemory and then in default cache (redis)
+    #   Invalidation is not necessary as the builds don't get updated
+    cache = tiered_django_cache([
```

Comment:
just thinking this should probably be declared outside the function so it's not re-created on every call

Reply:
Does this incur a cost to redis? I think those are pooled by the client, so it shouldn't be an issue, no?

Reply:
no cost to redis, just seems unnecessary work